engine/src/main/java/org/terasology/rendering/dag/RenderTaskListGenerator.java

```diff
@@ -90,7 +90,7 @@ private void generateTaskList() {
                             taskList.add(stateChange.generateTask());
                             persistentStateChanges.remove(stateChange.getClass());
 
-                        } else if (!stateChange.isEqualTo(persistentStateChange)) { // another new method, just for readability
+                        } else if (!stateChange.equals(persistentStateChange)) { // another new method, just for readability
```

Comment:
We don't need the comment anymore here! 

And good job on implementing the equals() method!